parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -472,7 +472,7 @@ loop = do
           if b then do
             respond (ShowDiff input (Branch.namesDiff destb merged))
             patch <- getPatchAt defaultPatchPath
-            void $ propagatePatch inputDescription patch dest
+            void $ propagatePatch input inputDescription patch dest
```

Comment:
`input` was piped around here and elsewhere for displaying the `NothingTodo` case.